library/modules/bigip_snat_pool.py

```diff
@@ -118,26 +139,34 @@
     from ansible.module_utils.network.f5.common import f5_argument_spec
     from ansible.module_utils.network.f5.common import transform_name
     from ansible.module_utils.network.f5.ipaddress import is_valid_ip
+    from ansible.module_utils.network.f5.compare import cmp_str_with_none
 
 
 class Parameters(AnsibleF5Parameters):
     api_map = {}
 
     updatables = [
-        'members'
+        'members',
+        'description',
     ]
 
     returnables = [
-        'members'
+        'members',
+        'description',
     ]
 
     api_attributes = [
-        'members'
+        'members',
+        'description',
     ]
 
 
 class ApiParameters(Parameters):
-    pass
+    @property
```

Comment:
this is not necessary unless the API returns 'none' in the value for when the value should be empty, if that is not the case you can remove it